src/main/generic/api/Configuration.js

```diff
@@ -255,7 +255,7 @@ Client.ConfigurationBuilder = class ConfigurationBuilder {
     }
 
     _requiredType(val, name, type) {
-        if (!val) throw new Error(`${name} is required`);
+        if (typeof val === 'undefined') throw new Error(`${name} is required`);
```

Comment:
I'd say `val === undefined` would be better, as we're not talking about the type (we have a type check in the next line)

Reply:
There's really no difference to the outcome with that change. I did it this way out of habit, because before ES2015 `undefined` could be re-assigned and back then it was good practice to use the `typeof` method.

However, I am fine with using your suggestion, but I would even prefer to remove this line completely and just leave the next line that checks for the type.

Reply:
I'd also be fine to remove this line completely in favor of the next one, if the error message in next line was changed to make it obvious that it was undefined (change to sth like `Type of ${name} must be ${type}, but is ${typeof val}`)

Reply:
Cool, done that! :+1: